docs/user-guide/connections.md

```diff
@@ -49,11 +49,11 @@ spec:
     - server:
         port: server-3/enp2s1
       switch:
-        port: s5248-01/Ethernet3
+        port: s5248-01/E1/1
     - server:
         port: server-3/enp2s2
       switch:
-        port: s5248-01/Ethernet4
+        port: s5248-01/E1/1
```

Comment:
should be different from the above as it is on the same switch, E1/2 for example?